programs/fuzz/common.c

```diff
@@ -64,6 +64,9 @@ int dummy_random( void *p_rng, unsigned char *output, size_t output_len )
     if( p_rng != NULL ) {
         //use mbedtls_ctr_drbg_random to find bugs in it
         ret = mbedtls_ctr_drbg_random(p_rng, output, output_len);
+    } else {
+        //fall through to pseudo-random
+        ret = 0;
```

Comment:
Not requesting change, just a note: this duplicates line 73, we could have used the pattern
```
#if COMPILE_TIME_CONDITION_FOR_X
if( run_time_condition_for_x ) {
   /* code using x */
}
else
#endif  
{
    /* code without x */
}
```
Here "code without x" is just one line, but when it's longer we want to avoid duplicating it.